examples/textConnect/contents/setting.html

```diff
@@ -4,210 +4,214 @@
  *
  * Licensed under the MIT License
 -->
-<div>  
-  <div class="block">
-    <label class="kintoneplugin-label">
-        <span id ="container_label">結合する項目</span>
-    </label><br >
-    <div class="kintoneplugin-desc">結合する項目を選択してください。（最大5つまで）</div>
-    
-    <div class="kintoneplugin-row" id="row1">
-        <span class="num">①</span>
-        <div class="kintoneplugin-select-outer">
-          <div class="kintoneplugin-select">
-            <select name="select1" id= "select1">
-              <option value="">-----</option>
-            </select>
-          </div>
-        </div>
-        <span class="plus">＋</span>
-        <div class="kintoneplugin-select-outer">
-          <div class="kintoneplugin-select">
-            <select name="select2" id= "select2">
-              <option value="">-----</option>
-            </select>
-          </div>
-        </div>
-        <span class="plus">＋</span>
-        <div class="kintoneplugin-select-outer">
-          <div class="kintoneplugin-select">
-            <select name="select3" id= "select3">
-              <option value="">-----</option>
-            </select>
-          </div>
-        </div>
-        <span class="plus">＋</span>
-        <div class="kintoneplugin-select-outer">
-          <div class="kintoneplugin-select">
-            <select name="select4" id= "select4">
-              <option value="">-----</option>
-            </select>
-          </div>
-        </div>
-        <span class="plus">＋</span>
-        <div class="kintoneplugin-select-outer">
-          <div class="kintoneplugin-select">
-            <select name="select5" id= "select5">
-              <option value="">-----</option>
-            </select>
-          </div>
-        </div>
+<div id="cf-plugin">
+  <div>
```

Comment:
Un-needed extra `<div>` and closing `</div>` later on

Reply:
No action was taken on my side as it seemed to be resolved.
I have also addressed the issue where there are duplicated ids.